dapp/src/constants/staking.ts

```diff
@@ -1,4 +1,4 @@
 // TODO: Read the value from the SDK, once we expose it
 export const MINIMUM_BALANCE = BigInt(String(5e6)) // 0.05 BTC
 
-export const SEASON_CAP = BigInt(String(5e10)) // 500 BTC
+export const TOTAL_VALUE_LOCKED_CAP = 1250 // 1250 BTC
```

Comment:
Non-blocking comment: what do you think about passing this value as an env variable? We could change the cap w/o updating the code.

Reply:
We need to consider the steps when we update the cap. Please see https://github.com/thesis/acre/pull/823#discussion_r1827782305

If we change just the cap, without updating the steps, we won't maintain the historic goals.

Reply:
Discussed with Kuba. Decided to hardcode it as a constant. It won't be changed that often.